app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,76 @@
+class CartsController < ApplicationController
+  before_action :find_product,
+                :find_item,
+                only: %i(
+                  create
+                  destroy
+                  update_cart
+                  update_rating_item
+                )
```

Comment:
chuyển sang dùng except cho gọn code nha em
```suggestion
                except: :index
```